assets/base.css

```diff
@@ -1540,26 +1612,35 @@ details[open] > .share-button__fallback {
   color: rgba(var(--color-foreground), 0.75);
 }
 
-.field__input:hover,
-.select__select:hover,
-.customer .field input:hover,
-.customer select:hover,
-.localization-form__select:hover {
-  box-shadow: 0 0 0 calc(var(--inputs-border-width) + 1px) rgba(var(--color-foreground), var(--inputs-border-opacity));
+.field:hover.field:after,
+.select:hover.select:after,
+.select__select:hover.select__select:after,
+.customer .field:hover.field:after,
+.customer select:hover.select:after,
+.localization-form__select:hover.localization-form__select:after {
+  box-shadow: 0 0 0 calc(0.1rem + var(--inputs-border-width)) rgba(var(--color-foreground),var(--inputs-border-opacity));
+  outline: 0;
+  border-radius: var(--inputs-radius);
 }
 
 .field__input:focus,
 .select__select:focus,
 .customer .field input:focus,
 .customer select:focus,
+.localization-form__select:focus.localization-form__select:after {
+  box-shadow: 0 0 0 calc(0.1rem + var(--inputs-border-width)) rgba(var(--color-foreground));
+  outline: 0;
+  border-radius: var(--inputs-radius);
+}
```

Comment:
cc @melissaperreault @svinkle

Reply:
I understand the concern, and agree; in this instance the focus state may difficult to discern. However, this is still technically accessible as the focus ring contrasts highly against its contrasting area (black on white) an is at least `2px`.

I also understand the "box within a box" concern as `:focus-visible` does not apply to `input` elements, therefore sighted mouse users will see the focus ring, causing the effect.

Since the locale/currency are native `summary`/`button` controls, we should be able to apply the `:focus-visible` styling, removing the "box within a box" affect for mouse/touch users. Native `input`s are the exception as focus styles will always display in these instances.

The inconsistent focus styling seems to be seen most prominently on Sense. The focus styles here are particularly difficult to discern due to low contrast. Perhaps there's an opportunity to revisit focus ring styles?

<details>
  <summary>Sense Demo</summary>

<video src="https://user-images.githubusercontent.com/1392632/149366481-da226a9b-e458-4f21-870c-bb82758c71fe.mp4" aria-label="Sense form showcasing low contrast focus rings." controls>



</details>

Reply:
Yeah I agree that in some situations, based on the settings it's just hard to tell where the focus is: [video](https://screenshot.click/22-01-o9snn-lqzfh.mp4)

Maybe the box-shadow can be slightly bigger than `0.1rem` 🤔